controllers/library.ts

```diff
@@ -49,10 +49,12 @@ const get_user_library = ({
         ? await db
             .knex("library")
             .join("album", "library.content_id", "album.id")
+            .join("artist", "artist.id", "album.artist_id")
             .select(
               "album.id as id",
               "album.created_at as createdAt",
               "album.artist_id as artistId",
+              "artist.name as artistName",
```

Comment:
```suggestion
              "artist.name as artist",
```

We alias this `artist` in other responses, should probably do the same here